server.py

```diff
@@ -832,6 +832,12 @@ async def skipped_boss_relics(ctx: ContextType, save: Savefile): # JSON_FP_PROP
 
     await ctx.send(" ".join(msg))
 
+@with_savefile("bottled")
+async def bottled_cards(ctx: ContextType, save: SaveFile):
+    """List all bottled cards"""
+    cards = save.bottled_cards
+    await ctx.send(", ".join(cards) if len(cards) > 0 else "We do not have any bottled cards.")
```

Comment:
Feel free to change the verbiage or logic however you think is better pattern-wise or anything